app/components/progress-bar.hbs

```diff
@@ -0,0 +1,51 @@
+<div ...attributes class='progress-bar-container'>
```

Comment:
- if we're following BEM convention should we change this to `progress-bar__container` ?

Reply:
made required changes.